src/modules/session/api.ts

```diff
@@ -65,6 +67,21 @@ export const extendedApi = api.injectEndpoints({
         body,
       }),
     }),
+
+    sendVerificationEmail: build.query<EmptyResponse, Request2FACode>({
+      query: () => ({
```

Comment:
Looks good, but you'll also need to pass the `email` argument here. E.g:

```
query: ({ email }) => ({
  url: "v1/auth/code",
  method: "GET",
  params: { email },
}),
```
should work